src/components/RocketElement.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getRockets } from '../redux/rockets/rockets';
+import SingleRocket from './SingleRockets';
+
+const RocketElement = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets);
+  useEffect(() => {
+    dispatch(getRockets());
+  }, []);
```

Comment:
- [ ] Please make sure that every time a component renders again, it doesn't fetch rockets or missions if they have already been fetched one time.

```
Actions should be dispatch once
Data should not be added to the store on every re-render (i.e. when changing views / using navigation).
```

![only-one-fetch](https://user-images.githubusercontent.com/79114233/141537596-6b5ba825-7d38-460b-aa9f-6e7ef6df4f92.gif)

Reply:
### Hello @D4nielJ ,
Thank you for your feedback.
The issue has just been fixed. Every time a component renders, it doesn't fetch rockets or missions again.

Reply:
Done ✔️